[2025-08-03T10:57:14.885+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:56:00+00:00/task_id=validate_model_performance permission to 509
[2025-08-03T10:57:14.886+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:56:00+00:00/task_id=validate_model_performance permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:56:00+00:00/task_id=validate_model_performance'
[2025-08-03T10:57:14.938+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:56:00+00:00/task_id=validate_model_performance permission to 509
[2025-08-03T10:57:14.940+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:56:00+00:00/task_id=validate_model_performance permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:56:00+00:00/task_id=validate_model_performance'
[2025-08-03T10:57:15.027+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_predict_to_db.validate_model_performance scheduled__2025-08-03T10:56:00+00:00 [queued]>
[2025-08-03T10:57:15.046+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_predict_to_db.validate_model_performance scheduled__2025-08-03T10:56:00+00:00 [queued]>
[2025-08-03T10:57:15.047+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-08-03T10:57:15.067+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): validate_model_performance> on 2025-08-03 10:56:00+00:00
[2025-08-03T10:57:15.078+0000] {standard_task_runner.py:57} INFO - Started process 1155 to run task
[2025-08-03T10:57:15.083+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'train_predict_to_db', 'validate_model_performance', 'scheduled__2025-08-03T10:56:00+00:00', '--job-id', '105', '--raw', '--subdir', 'DAGS_FOLDER/train_predict_to_db_dag.py', '--cfg-path', '/tmp/tmp0ptux45k']
[2025-08-03T10:57:15.087+0000] {standard_task_runner.py:85} INFO - Job 105: Subtask validate_model_performance
[2025-08-03T10:57:15.161+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:56:00+00:00/task_id=validate_model_performance permission to 509
[2025-08-03T10:57:15.161+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:56:00+00:00/task_id=validate_model_performance permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:56:00+00:00/task_id=validate_model_performance'
[2025-08-03T10:57:15.164+0000] {task_command.py:416} INFO - Running <TaskInstance: train_predict_to_db.validate_model_performance scheduled__2025-08-03T10:56:00+00:00 [running]> on host 407839faddcf
[2025-08-03T10:57:15.281+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='train_predict_to_db' AIRFLOW_CTX_TASK_ID='validate_model_performance' AIRFLOW_CTX_EXECUTION_DATE='2025-08-03T10:56:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-03T10:56:00+00:00'
[2025-08-03T10:57:15.336+0000] {python.py:194} INFO - Done. Returned value was: {'mse': 1277355.499056128, 'rmse': 1130.201530283926, 'mae': 888.6124708387587, 'r2': 0.901157297523326}
[2025-08-03T10:57:15.374+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=train_predict_to_db, task_id=validate_model_performance, execution_date=20250803T105600, start_date=20250803T105715, end_date=20250803T105715
[2025-08-03T10:57:15.417+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-08-03T10:57:15.441+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
